### ISSUE TITLE
Standalone ecleaner v2.5

### ISSUE THREAD

**Issue by delicacy1** [diegocr/PrefMon#6]
That would be so cool to obtain it since i don't use the rest at all.

**Comment by tunguyen6** [comment]
I agree. It is here, https://addons.mozilla.org/en-US/firefox/addon/ecleaner/, but discontinued.
